policyengine_canada/parameters/gov/provinces/sk/tax/income/credits/dividend_tax_credit/eligible_cal_percent.yaml

```diff
@@ -0,0 +1,12 @@
+description: Saskatchewan applies this weight percentage on the eligible taxable amount of dividends in calculating the total dividend tax credit.
```

Comment:
I think this is for "other than eligible taxable amount of dividends" instead of "eligible". According to this [web](https://www.canada.ca/en/revenue-agency/services/tax/individuals/topics/about-your-tax-return/tax-return/completing-a-tax-return/personal-income/line-12000-taxable-amount-dividends-eligible-other-than-eligible-taxable-canadian-corporations.html), line 12010 (which is B of your [first reference](https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/5008-d/5008-d-22e.pdf#page=3)) is "other than eligible".